src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> stringsList;
+        try {
+            stringsList = Files.readAllLines(new File(fileName).toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Text read failed!", e);
+        }
+
+        StringBuilder stringBuilder = new StringBuilder();
+        for (String string : stringsList) {
+            stringBuilder.append(string.toLowerCase()).append(" ");
+        }
+
+        String[] arrayText = stringBuilder.toString().split(" ");
+        List<String> readFiles = new ArrayList<>();
```

Comment:
the variable names are confusing - let's reconsider them

Reply:
Done. =)